scripts/simple_cors_http_server.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+from http.server import HTTPServer, SimpleHTTPRequestHandler, test
+import sys
+
+
+class CORSRequestHandler(SimpleHTTPRequestHandler):
+    def end_headers(self):
+        self.send_header("Access-Control-Allow-Origin", "*")
+        SimpleHTTPRequestHandler.end_headers(self)
+
+
+if __name__ == "__main__":
+    test(
+        CORSRequestHandler,
+        HTTPServer,
+        port=int(sys.argv[1]) if len(sys.argv) > 1 else 8000,
+    )
```

Comment:
Why is this needed?

Reply:
The renderer needs access to the files used for image mobjects in order to render them but I don't know how to configure it to access local files without [disabling webSecurity](https://stackoverflow.com/questions/48777336/is-disabling-websecurity-on-the-electron-window-the-only-way-to-display-local-im) which is considered bad practice. So I used this to serve the files from a local server. It may not be necessary but I don't know of another safe way to handle ImageMobjects.

Reply:
[This](https://github.com/ManimCommunity/manim-renderer/blob/master/src/App.vue#L314) is where it's handled by the renderer.